com.unity.render-pipelines.universal/Runtime/ForwardRendererData.cs

```diff
@@ -39,20 +40,45 @@ public sealed class ShaderResources
 
             [Reload("Shaders/Utils/MaterialError.shader")]
             public Shader materialErrorPS;
+
+            // Core blitter shaders, adapted from HDRP
+            // TODO: move to core and share with HDRP
+            [Reload("Shaders/Utils/CoreBlit.shader"), SerializeField]
+            internal Shader coreBlitPS;
+            [Reload("Shaders/Utils/CoreBlitColorAndDepth.shader"), SerializeField]
+            internal Shader coreBlitColorAndDepthPS;
+
+
+            [Reload("Shaders/CameraMotionVectors.shader")]
+            public Shader cameraMotionVector;
+
+            [Reload("Shaders/ObjectMotionVectors.shader")]
+            public Shader objectMotionVector;
```

Comment:
Why are these added
And do you have a JIRA for that TODO:?

Reply:
These are coped as is from the `UniversalRendererData.cs` to make sure we don’t loose any user data as the existing UniversalRenderDatas will briefly become ForwardRendererDatas.